packages/grid/_modules_/grid/hooks/features/rows/useGridEditRows.ts

```diff
@@ -341,4 +343,12 @@ export function useGridEditRows(
       changeEvent: GRID_EDIT_ROWS_MODEL_CHANGE,
     });
   }, [apiRef, props.editRowsModel, props.onEditRowsModelChange]);
+
+  React.useEffect(() => {
+    const oldEditRowsModel = apiRef.current.getState().editRows;
```

Comment:
How about we reverse the dichotomy. What we call old is the current value, 

```suggestion
    const currentEditRowsModel = apiRef.current.getState().editRows;
```